web/api/web_api_v1.c

```diff
@@ -1250,6 +1271,7 @@ static struct api_command {
 #if defined(ENABLE_ML)
         { "anomaly_events",     0, WEB_CLIENT_ACL_DASHBOARD, web_client_api_request_v1_anomaly_events     },
         { "anomaly_event_info", 0, WEB_CLIENT_ACL_DASHBOARD, web_client_api_request_v1_anomaly_event_info },
+        { "ml_info",            0, WEB_CLIENT_ACL_DASHBOARD, web_client_api_request_v1_ml_info            },
```

Comment:
Do you think we should support at least `ml_info` if ml has not been compiled? It could perhaps return a relevant `error` in such case? (Again depends on who is consuming this endpoint, is it the cloud? Or just informational) ?

Reply:
No, consumers should consult `/info` to check if `ml_info` is available.